main.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'screened'
+require_relative 'morning_session'
+require_relative 'afternoon_session'
+
+class Conference
+  def initialize(talks)
+    @talks = talks
+    @track1 = []
+    @track2 = []
+  end
+
+  def load_sessions
+    morning_session
+    afternoon_session
+  end
+
+  def morning_session
```

Comment:
Method names should be an action or verb. It can be a noun only when the return value is the method name.

Reply:
Done